### DISCUSSION TITLE
[Feature Request] Enable hierarchical table view

### DISCUSSION THREAD

**Discussion by lucaswhornsby** [grafana/grafana#57845] [Dashboards, Panels & Visualizations]
Building off this post in Grafana Community (https://community.grafana.com/t/hierarchy-in-grafana-table/53336), I would like to request the option to configure tables with a hierarchy such that individual rows can be comprised by sub-rows that can be expanded, and so forth. For an illustration, see below screenshot (from the original Grafana Community post). 

![image](https://user-images.githubusercontent.com/113460677/198706507-7ecb8252-d999-4a14-bfeb-5f446bd60134.png)

This feature would enable drill-down analysis that can bring significant value to multiple diverse Grafana use cases.

**Comment by lucaswhornsby** [comment]
Hello @zuchka ! I created this as a follow up to [this post](https://community.grafana.com/t/hierarchy-in-grafana-table/53336) in the Grafana Labs forum! Could you please provide an update on the status of this request? Is this in Grafana's roadmap? Do you have any updated recommendations on implementation? Thanks :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by KevMyst** [comment]
I would also like to ask for the option to configure tables with hierarchy in such a way that individual rows can be made up of sub-rows that can be expanded, and so on, as in the first post.

**Comment by gelicia** [comment]
Due to continuing interest in this, I have migrated this discussion. The issue is #83415
